x_net.py

```diff
@@ -105,10 +105,10 @@ def kronecker_emr_network(radix_lists, B):
   num_layers = len(emr_layers)
   emr_shape, emr_paths, emr_connections_per_neuron = info['shape'], info['paths'],\
       info['connections_per_neuron']
-  shape = [emr_shape[i] * B[i] for i in range(len(shape))]
+  shape = [emr_shape[i] * B[i] for i in range(len(emr_layers))]
   # the first and last numbers in B do not add paths, but increase the
   # number of input and output neurons.
-  paths *= np.prod(B[1:-1]) 
+  paths = np.prod(B[1:-1]) 
```

Comment:
Ryan, why did you replace this here? I thought to give the total number of paths, you have multiply prod(B[1:-1} ) with prod(N[1:])

Reply:
Yeah, what I did was a mistake. I think I commented something else out which made the "*=" return an error, so I changed that line before I recognized I should have commented it all out. Sorry about the confusion.

Would you be willing to make me an official collaborator for this repo? Whenever I migrate stuff over, I'll be sure to make it as a pull request so that it doesn't interfere with any of your changes.

Reply:
Oh yeah for sure